cookiecutter.json

```diff
@@ -10,5 +10,25 @@
     "use_flower": "{% if cookiecutter.use_celery == 'y' %}y{% else %}n{% endif %}",
     "celery_flower_user": "{{(['QWERTYUIOPASDFGHJKLZXCVBNMqwertyuiopasdfghjklzxcvbnm1234567890']*16)|map('random')|join('')}}",
     "celery_flower_password": "{{(['QWERTYUIOPASDFGHJKLZXCVBNMqwertyuiopasdfghjklzxcvbnm1234567890']*16)|map('random')|join('')}}",
-    "sentry_dsn": "https://<key>@sentry.io/<project>"
+    "sentry_dsn": "https://<key>@sentry.io/<project>",
+    "https_redirect": "n",
+    "https_proxy_header": "X-Scheme",
+    "csp_enabled": "y",
```

Comment:
ask below only `if csp_enabled == 'y'`

Reply:
I am not sure how to do it, do you have an idea?

Reply:
Sorry, just realized it's json. Please ignore this comment